bridges/Rule34pahealBridge.php

```diff
@@ -8,12 +8,14 @@ class Rule34pahealBridge extends Shimmie2Bridge {
 	const URI = 'https://rule34.paheal.net/';
 	const DESCRIPTION = 'Returns images from given page';
 
+	const PATHTODATA = '.shm-thumb';
+
 	protected function getItemFromElement($element){
 		$item = array();
-		$item['uri'] = $this->getURI() . $element->href;
+		$item['uri'] = rtrim($this->getURI(), '/') . $element->find('.shm-thumb-link', 0)->href;
```

Comment:
There is defaultLinkTo for fixing URLs. But it looks like it will be more complicated to use it https://github.com/RSS-Bridge/rss-bridge/blob/ff98efe8dc13c978be3a715da44f825af194513c/lib/html.php#L96-L107